patches/0078-ramips-yuncore-ax820-increase-certificate-parition-to-512kB.patch

```diff
@@ -0,0 +1,47 @@
+From 62ceb35f69772f625d6e3c0f304ff6ddf1c4109a Mon Sep 17 00:00:00 2001
+From: HaiBac <mail@bacnh.com>
+Date: Fri, 12 Apr 2024 09:06:11 +0000
+Subject: [PATCH] ramips: yuncore-yuncore_ax820 increase certificate parition to 512kB
+
+Signed-off-by: HaiBac <mail@bacnh.com>
+---
+ target/linux/ramips/dts/mt7621_yuncore_ax820.dts | 6 +++---
+ target/linux/ramips/image/mt7621.mk              | 2 +-
+ 2 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/target/linux/ramips/dts/mt7621_yuncore_ax820.dts b/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
+index b2f55b9be0..570a34d65e 100644
+--- a/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
++++ b/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
+@@ -120,12 +120,12 @@
+ 			partition@90000 {
+ 				compatible = "denx,uimage";
+ 				label = "firmware";
+-				reg = <0x90000 0xf60000>;
```

Comment:
will break all ax820 boards currently deployed

Reply:
All ax820 boards currently deployed - `certificates` folder is not mounted properly - instead at `overlay` partition.

`check_certificates` is called at first, so if there is certs inside `/certificates` folder then no mount mechanism run, thus this will not break ax820 currently deployed.

Reply:
changing the size will break, people currently deploy squashfs 64k block in the partition

Reply:
got it - let me revise back to 9k and test more on this device.